sdk-libs/compressible-client/wrapper.md

```
@@ -0,0 +1,1013 @@
+│                                                                        │
+│  2. Program PDAs + CTokens: All in same tx if same program             │
+│     - decompress_accounts_idempotent(pda1, pda2, ctoken1, ctoken2)     │
+│     - Order: PDAs first, then CTokens (CPI context handling)           │
+│                                                                        │
+│  3. Mixed programs: Separate transactions                              │
+│     - Each program_id gets its own decompress_accounts_idempotent      │
+│                                                                        │
+│  4. Compute limits: May need to split large batches                    │
+│     - ~200k CU per decompression                                       │
+│     - ~1.4M CU limit per tx                                            │
+│     - Max ~7 decompressions per tx                                     │
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)